pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -105,6 +107,28 @@ type OperationExecutor interface {
 	// actual state of the world to reflect that.
 	UnmountDevice(deviceToDetach AttachedVolume, actualStateOfWorld ActualStateOfWorldMounterUpdater, mounter mount.Interface) error
 
+	// MapVolume is used when the volumeMode is 'Block'.
+	// This method creates a symbolic link to the volume from both the pod
+	// specified in volumeToMount and global map path.
+	// Specifically it will:
+	// * Wait for the device to finish attaching (for attachable volumes only).
+	// * Update actual state of world to reflect volume is globally mounted/mapped.
+	// * Map volume to global map path using symbolic link.
+	// * Map the volume to the pod device map path using symbolic link.
+	// * Update actual state of world to reflect volume is mounted/mapped to the pod path.
+	MapVolume(waitForAttachTimeout time.Duration, volumeToMount VolumeToMount, actualStateOfWorld ActualStateOfWorldMounterUpdater) error
```

Comment:
@jingxu97 @msau42 

Comments picking up from #54752.
```
jingxu97 15 hours ago  Member
do we need to make it public. Seems like all MountVolume or MapVolume in operation executor could be private?
```
I believe we could make all methods in OperationExecutor interfaceas as private since they always used via operation executor, but I think we shouldn't change all of them in this one PR.
Related methods are following. I agree on making them as private in this PR.

- MountVolume, MapVolume
- UnmountVolume, UnmapVolume
- UnmountDevice, UnmapDevice